examples/computer_vision/cifar10_tf_keras/const.yaml

```diff
@@ -19,4 +17,5 @@ searcher:
     epochs: 32
 min_validation_period:
   epochs: 1
+max_restarts: 0
```

Comment:
while I do this on every example _I_ run, I don't think we should be setting this in our examples.

Though I'm not opposed to making it the default... I've always found this deeply annoying.

Reply:
Removed.  I also do this on every example I run, so I do think it should be the default.